e2e/tests/nightly/gke-apps/cromwell.spec.ts

```diff
@@ -66,29 +76,71 @@ describe('Cromwell GKE App', () => {
 
     await appsPanel.pollForStatus(expandedCromwellXpath, 'Running', 15 * 60e3);
 
-    const deleteXPath = `${expandedCromwellXpath}//*[@data-test-id="Cromwell-delete-button"]`;
-    const deleteButton = new Button(page, deleteXPath);
-    expect(await deleteButton.exists()).toBeTruthy();
-    await deleteButton.click();
+    const isDeleted = cromwellPanel.deleteCromwellGkeApp();
+    expect(isDeleted).toBeTruthy();
+  });
+
+  test('Run cromwell using python notebook', async () => {
+    // Create or re-use workspace
+    await findOrCreateWorkspace(page, { workspaceName });
```

Comment:
we should use a separate workspace for this, so they don't interfere with each other